js/src/tooltip.js

```diff
@@ -569,10 +576,10 @@ class Tooltip extends BaseComponent {
 
   _fixTitle() {
     const title = this._element.getAttribute('title')
-    const originalTitleType = typeof this._element.getAttribute('data-bs-original-title')
+    const originalTitleType = typeof Manipulator.getDataAttribute(this._element, 'original-title')
```

Comment:
I am not sure I have understand correctly your thoughts, but it calls the below 

```js 


  getDataAttribute(element, key) {
    return normalizeData(element.getAttribute(`data-bs-${normalizeDataKey(key)}`))
  },
```

Reply:
Yeah, exactly :) That's not the same as `this._element.getAttribute`. And it seems we already use `Element.getAttribute()` in more places anyway.

Reply:
Since the manipulator methods normalize the value that is not required here. Only the string values are coming from the attributes that do not require normalization. :thinking: 

_But keeping Manipulator methods everywhere in the code makes consistency IMO. (That should be done in a separate PR though)_

Reply:
Then, these changes can be considered in a new PR. :thinking:

Reply:
I have revert them all